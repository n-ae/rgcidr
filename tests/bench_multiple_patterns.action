-f tests/patterns_bench.txt
