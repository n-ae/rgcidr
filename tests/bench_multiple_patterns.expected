192.168.1.1
192.168.1.2
10.0.0.1
10.0.0.2
172.16.0.1
172.16.0.2
203.0.113.1
203.0.113.2
198.51.100.1
198.51.100.2
192.168.2.1
192.168.2.2
10.1.0.1
10.1.0.2
172.17.0.1
172.17.0.2
10.0.0.100
10.0.0.200
10.0.1.1
10.0.1.2
